src/Microsoft.DotNet.Cli.Utils/CommandResolution/PublishedPathCommandResolver.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class PublishedPathCommandResolver : ICommandResolver
```

Comment:
Maybe "ApplicationCommandResolver"??  "DotNetAppCommandResolver"??  

This can run any .NET Core app, right?

I might be reaching here...

---
In reply to: [76478100](https://github.com/dotnet/cli/pull/4102#discussion_r76478100) [](ancestors = 76478100)